src/main/java/org/liquidengine/legui/system/renderer/nvg/util/NvgShapes.java

```diff
@@ -39,6 +39,9 @@ private NvgShapes() {
      * @param bgColor rectangle background color.
      */
     public static void drawRect(long nvg, Vector2fc position, Vector2fc size, Vector4fc bgColor) {
+        if (bgColor.w() == 0) {
```

Comment:
since bgColor.w() is a float, maybe is better to use `Math.abs(bgColor.w()) < MIN_ALPHA` with MIN_ALPHA as 0.001 or something like that

Reply:
probably yes.